app/models/detector/suggested_resource.rb

```diff
@@ -76,5 +76,9 @@ def self.bulk_replace(input)
         record.save
       end
     end
+
```

Comment:
Would you mind adding docs for this method and the others that don't have them in this class?

See https://github.com/MITLibraries/tacos/blob/main/app/models/detector/journal.rb#L19-L32 for syntax examples that work well with our documentation server you can run via `make docserver`